src/query/external/useSEContractVersionHistory.ts

```diff
@@ -38,7 +40,9 @@ export const useSEContracVersionHistory = ({
         throw `Something went wrong. ${e}`;
       }
     },
-    enabled: Boolean(networkId && contractId),
+    enabled: Boolean(isActive && networkId && contractId),
+    // Keep data for 30 seconds
```

Comment:
is this keep data for 30 seconds or don't call another api request for 30 seconds?

Reply:
Yes, that's precisely what it means. I thought adding a little "cache" time for cases when users quickly go through tabs was good. This way, we won't be making too many API calls.